examples/datasets/plot_brainstorm_auditory_data.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+"""
+====================================
+Brainstorm auditory tutorial dataset
+====================================
+
+Here we compute the evoked from raw for the auditory Brainstorm
+tutorial dataset. For comparison, see:
+http://neuroimage.usc.edu/brainstorm/Tutorials/Auditory
+
+References
+----------
+.. [1] Tadel F, Baillet S, Mosher JC, Pantazis D, Leahy RM.
+       Brainstorm: A User-Friendly Application for MEG/EEG Analysis.
+       Computational Intelligence and Neuroscience, vol. 2011, Article ID
+       879716, 13 pages, 2011. doi:10.1155/2011/879716
+"""
+
+# Authors: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+# TODO:
+# - figure out why there is contamination at ~350 ms in standard condition
+# - complete source localization
+
+import numpy as np
+from scipy.io import loadmat
+
+import mne
+from mne.datasets.brainstorm import bst_auditory
+from mne.io import Raw
+
+print(__doc__)
+
+data_path = bst_auditory.data_path()
+
+tmin, tmax = -0.1, 0.5
+event_id = dict(standard=1, deviant=2)
+reject = dict(mag=4e-12, eog=250e-6)
+subject = 'bst_auditory'
+subjects_dir = data_path + '/subjects'
+raw_fnames = [data_path + '/MEG/bst_auditory/S01_AEF_20131218_0%s_raw.fif'
+              % run_num for run_num in range(1, 3)]
+erm_fname = data_path + '/MEG/bst_auditory/S01_Noise_20131218_01_raw.fif'
+
+raw = Raw(raw_fnames, preload=True, add_eeg_ref=False)
+raw_erm = Raw(erm_fname, preload=True, add_eeg_ref=False)
+
+# set EOG channel
+raw.set_channel_types({'HEOG': 'eog', 'VEOG': 'eog', 'ECG': 'ecg'})
+
+# parse Brainstorm bad event data
```

Comment:
to me the handling of bad segments should be done at the raw level and the
epochs should be aware of it. It's much cleaner.

the bad segment should also be visible from the raw viewer.